### PR TITLE
add more detail / clarification to conflict-of-interest guidelines

### PR DESCRIPTION
**Pull request by ultrasaurus**
adding clarifications to streamline conflict-of-interest review/approval
addresses https://github.com/cncf/sig-security/issues/341

### CODE REVIEW

File: assessments/guide/security-reviewer.md
Comment (JustinCappos) [inline assessments/guide/security-reviewer.md]:
```suggestion
* A hard conflict makes a reviewer ineligible to review a project.
```

File: assessments/guide/security-reviewer.md
Comment (JustinCappos) [inline assessments/guide/security-reviewer.md]:
```suggestion
* In some cases, one or more reviewers may have a soft conflict.  This may be permissible so long
```
Change is moving plural to singular for clarity.

File: assessments/guide/security-reviewer.md
Comment (JustinCappos) [inline assessments/guide/security-reviewer.md]: I think the lead being aware is good, but this does also seem to mildly conflict with the statement below about the facilitator / chairs reviewing the conflict of interest.  Do we want the lead assessor to have the final say here?  I'd prefer it was centralized across reviews.

File: assessments/guide/security-reviewer.md
Comment (JustinCappos) [inline assessments/guide/security-reviewer.md]:
```suggestion
  3. they believe that the review team is able to provide a balanced and fair assessment
```

File: assessments/guide/security-reviewer.md
Comment (JustinCappos) [inline assessments/guide/security-reviewer.md]: This is for soft conflicts, I assume.  Can hard conflicts ever be waived?  Can a conflicted person still participate unofficially?

File: assessments/guide/security-reviewer.md
Comment (ultrasaurus) [reply assessments/guide/security-reviewer.md]: There's been discussion that we really want someone on the review team that has experience using the project, yet that is a soft conflict, so I was trying to word this to make clear that we're actually *encouraging* at least one person with a soft conflict.  I didn't want to make a long section even longer, but maybe it would be good to just say that: 

> Typically, at least one reviewer will have a soft conflict (since the review will benefit from the perspective of someone experienced with the project).  This is permissible so long...

What do you think?

File: assessments/guide/security-reviewer.md
Comment (ultrasaurus) [reply assessments/guide/security-reviewer.md]: The above section is attempting to make clear that individual reviewers must make sure they document a conflict-of-interest if it happens after initial approval (which happened a while back via acquisition / merger) and its not enough to just comment on the issue, they have to also make sure the lead security reviewer acknowledges their note -- of course that may entail wider review depending on the nature of the conflict.  Not sure how to make that more clear...

File: assessments/guide/security-reviewer.md
Comment (ultrasaurus) [reply assessments/guide/security-reviewer.md]: It's hard to imagine that we would waive a hard conflict, I'm open to writing down that we would disallow chair approval of such.

I think we say elsewhere that any SIG member (or community member?) can participate... would it be helpful to make it explicit here?  I don't want to imply that there is some kind of "unofficial reviewer" role... just cuz that feels complicated.  Maybe someone can suggest words :)

File: assessments/guide/security-reviewer.md
Comment (lumjjb) [reply assessments/guide/security-reviewer.md]: hmm, how about:

"It is not unusual for reviewers to have soft conflicts. The diversity of reviewers that are familiar with a project and can provide a deeper insight together with a fresh set of eyes is beneficial to the success of a security review."

File: assessments/guide/security-reviewer.md
Comment (lumjjb) [inline assessments/guide/security-reviewer.md]: consistency with bullets on top, caps or no caps

File: assessments/guide/security-reviewer.md
Comment (lumjjb) [inline assessments/guide/security-reviewer.md]: when reading these bullets, 'their'  and 'they' was a bit ambiguous (referring to reviewers or lead reviewers?) Since there was a mention in bullet 1 with "all reviewers"

File: assessments/guide/security-reviewer.md
Comment (TheFoxAtWork) [reply assessments/guide/security-reviewer.md:59]:
```suggestion
* It is not unusual for reviewers to have soft conflicts. The diversity of reviewers that are familiar with a project and can provide a deeper insight together with a fresh set of eyes is beneficial to the success of a security review.
```

File: assessments/guide/security-reviewer.md
Comment (TheFoxAtWork) [inline assessments/guide/security-reviewer.md:60]:
```suggestion
```

File: assessments/guide/security-reviewer.md
Comment (TheFoxAtWork) [inline assessments/guide/security-reviewer.md:61]:
```suggestion
```

File: assessments/guide/security-reviewer.md
Comment (TheFoxAtWork) [inline assessments/guide/security-reviewer.md:66]:
```suggestion
All reviewers must provide a conflict declaration to indicate which hard or soft conflicts do, or do not exist 
when they volunteer to be a reviewer.  This is done by placing a comment on the issue associated with the 
assessment.
Should a conflict arise during the time of the assessment, reviewers should notify the lead security reviewer when 
they become aware of the potential conflict, so the new conflict may be consulted with the facilitator and/or chairs.
```

File: assessments/guide/security-reviewer.md
Comment (TheFoxAtWork) [inline assessments/guide/security-reviewer.md:70]:
```suggestion
* Check that all reviewers have conflict-of-interest declarations,
```

File: assessments/guide/security-reviewer.md
Comment (TheFoxAtWork) [inline assessments/guide/security-reviewer.md]:
```suggestion
* Ask clarifying questions, if needed,
* Assert that:
  1. They have reviewed all conflict-of-interest declarations;
  2. There are no hard-conflicts; and
  4. They believe that the review team is able to provide a balanced and fair assessment
```

File: assessments/guide/security-reviewer.md
Comment (TheFoxAtWork) [inline assessments/guide/security-reviewer.md:86]:
```suggestion
be documented.
In most cases, the existence of a hard conflict will prevent a SIG member from participating in the assessment for 
which their hard conflict exists.  Depending on the circumstances of the particular conflict, the assessment, and the 
may project, the chairs and the assessment facilitator may determine if the hard conflict may be waived.  Should this 
occur, the decision's justification will be documented to ensure it clearly depicts the circumstances for granting the 
waiver.
```

File: assessments/guide/security-reviewer.md
Comment (TheFoxAtWork) [inline assessments/guide/security-reviewer.md:74]:
```suggestion
* Provide their own declaration of any potential conflict-of-interest (or lack thereof),
* Ask clarifying questions, if needed
* Assert that:
  1. They have reviewed all conflict-of-interest declarations from the reviewers;
  2. There are no hard-conflicts present without a waiver;
  3. They believe that the review team is able to provide a balanced and fair assessment
```

File: assessments/guide/security-reviewer.md
Comment (lumjjb) [inline assessments/guide/security-reviewer.md:82]: this includes soft conflicts right, this seems like it may be a potentially high overhead for chairs, since soft conflicts are likely to be common. How about making this discretionary to the security assessment facilitator

File: assessments/guide/security-reviewer.md
Comment (lumjjb) [inline assessments/guide/security-reviewer.md:89]: I think the 2 chair requirement should perhaps be for this?

File: assessments/guide/security-reviewer.md
Comment (TheFoxAtWork) [reply assessments/guide/security-reviewer.md:82]: So the way this is written, means that either the Security Assessment Facilitator Or a chair must review the entire assertion.  assertions include both hard and soft conflicts. I'll adjust the "any conflicts" to be "any hard conflicts, or multiple significant soft conflicts" for the 2 chair approval.

Comment (TheFoxAtWork) [inline assessments/guide/security-reviewer.md:82]:
```suggestion
If there any hard conflicts, or multiple significant soft conflicts are presented, then 2 SIG-Security chairs must approve
```

File: assessments/guide/security-reviewer.md
Comment (TheFoxAtWork) [inline assessments/guide/security-reviewer.md:87]:
```suggestion
project, two chairs and the assessment facilitator may determine if the hard conflict may be waived.  Should this 
```

File: assessments/guide/security-reviewer.md
Comment (TheFoxAtWork) [reply assessments/guide/security-reviewer.md:89]: corrected the wording, somehow it was miswritten, let me know if this works @lumjjb

**Comment by TheFoxAtWork** [comment]
@JustinCappos @dshaw @pragashj please review/approve

File: assessments/guide/security-reviewer.md
Comment (JustinCappos) [inline assessments/guide/security-reviewer.md:6]: If I recall, I've always done COI (or asked the chairs to).  I feel like pushing this to the ever-changing lead is a way to make this more uneven, which feels like the wrong thing to outsource especially since this is so little work...

File: assessments/guide/security-reviewer.md
Comment (TheFoxAtWork) [inline assessments/guide/security-reviewer.md:8]:
```suggestion
and recruiting additional reviewers as appropriate, and managing the documentation updates (self-assessment and 
```

File: assessments/guide/security-reviewer.md
Comment (TheFoxAtWork) [inline assessments/guide/security-reviewer.md:9]:
```suggestion
final assessment) to the project's assessment folder.
```

**Review by TheFoxAtWork — APPROVED** [review]

**Pull request merged**
